### ISSUE TITLE
Accessibility Voice Over Cannot Sort

### ISSUE THREAD

**Issue by UndercoverOp** [Calvin-LL/Reorderable#23]
Hello,

I have set up a LazyColumn and used ReorderableItem with .draggableHandle() on an Image within the Card that is inside the ReorderableItem, but I am unable to sort the items when voice over is enabled.

Is there already something existing for voice over sorting control or would you be able to allow a double click to active the ReorderableItem so the user can drag it then on release drop the item?

Let me know!

Thank you!

**Comment by Calvin-LL** [comment]
Any examples of how this is handled in other apps? Currently on a plane but I will be looking into this after my long flight.

**Comment by Calvin-LL** [comment]
I went around my Google Pixel phone to see how Google handles accessibility for drag and drop reordering.

1. In System > Languages > System Languages, if there are multiple languages, it shows a handle for drag and drop reordering for every item. The drag and drop handle is completely invisible to TalkBack. There are no additional actions in the TalkBack menu.
1. In Gboard Settings > Languages, after clicking the Edit button, it shows a handle for drag and drop reordering for every item. The drag and drop handle is completely invisible to TalkBack. There are no additional actions in the TalkBack menu.
1. In the Notification Drawer > Quick Settings > Edit Mode, there's a "Move tile" action in the TalkBack menu of every item in the grid, clicking it enables a mode where you can select another item in the grid to swap the current item with.

**Comment by Calvin-LL** [comment]
I think a good solution would be to add an action to the TalkBack menu to move an item up or down by swapping the selected item with the next reorderable item. I also think it should be up to the user of this library to add them (partly because of i18n, I can't translate "Move up" into all the languages) but I will add them to the example app.

**Comment by UndercoverOp** [comment]
Yeah I ended up adding a move up and move down option in the talk back menu which works good. In the future I might switch out the drag handle for and up and down buttons on the row itself to make it easier for a talk back user to sort items. Thank you though!

**Comment by Calvin-LL** [comment]
I have added "Move Up"/"Move Down"/"Move Left"/"Move Right" actions to the TalkBack menu for list items in the demo app and a section on Accessibility in the README.
